gp-templates/translations.php

```diff
@@ -105,7 +105,7 @@
 		$sort_values_only    = array_filter( $sort );
 		$filters_and_sort    = array_merge( $filters_values_only, $sort_values_only );
 		// Remove any non-string or non-numeric values from the array.
-		$filters_and_sort    = array_filter( $filters_and_sort, 'is_scalar' );
```

Comment:
Maybe we want to remove this change from this PR.

Reply:
Done. It was a change made automatically by the PHPCBF tool, but it is ok for me to remove it from this PR.